soh/src/code/audio_synthesis.c

```diff
@@ -653,7 +653,11 @@ Acmd* AudioSynth_DoOneAudioUpdate(s16* aiBuf, s32 aiBufLen, Acmd* cmd, s32 updat
     }
 
     updateIndex = aiBufLen * 2;
-    aInterleave(cmd++, DMEM_TEMP, DMEM_LEFT_CH, DMEM_RIGHT_CH, updateIndex);
+    if (CVarGetInteger("gMirroredWorld", 0)) {
+        aInterleave(cmd++, DMEM_TEMP, DMEM_RIGHT_CH, DMEM_LEFT_CH, updateIndex);
```

Comment:
Is it possible to remove this branch by doing math operations on CVarGetInteger and DMEM_RIGHT_CH and DMEM_LEFT_CH?